Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3_6_5.yml

```diff
@@ -2108,6 +2108,37 @@ inputs:
   required: false
   description: "Possible values: True/False.\nWhether to provide user verification for blocking IPs. \n\nFalse - No prompt will be displayed to the user.\nTrue - The server will ask the user for blocking verification and will display the blocking list."
   playbookInputQuery:
+inputSections:
+- inputs:
+  - LinkSimilarIncidents
+  - CriticalUsernames
+  - CriticalHostnames
+  - CriticalADGroups
+  - incident_id
+  - XDRDomain
+  name: Incident Management
+  description: Incident management settings and data, including escalation processes, user engagements and ticketing methods
+- inputs:
+  - InternalRange
+  - InternalDomainName
+  - InternalHostRegex
+  name: Enrichment
+  description: Enrichment settings and data, including assets and indicators enrichment using third-side enrichers.
```

Comment:
do you mean third-party (instead of third-side)

Reply:
done